net-irc/atheme-services/files/atheme-services.initd

```diff
@@ -0,0 +1,36 @@
+#!/sbin/openrc-run
+# Copyright 1999-2011 Gentoo Foundation
```

Comment:
I used 'repoman -dx full', but not 'repoman commit' because I don't yet have GPG keys set up.  Fixed.